internal/flypg/pg.go

```diff
@@ -606,3 +606,11 @@ func diskSizeInBytes(dir string) (uint64, error) {
 	}
 	return stat.Blocks * uint64(stat.Bsize), nil
 }
+
+func (*PGConfig) reload(ctx context.Context) error {
```

Comment:
This totally works, but you can also issue a reload using sql:  https://github.com/fly-apps/postgres-flex/blob/master/internal/flypg/admin/admin.go#L324-L329